notes/stories/literature/novel-various.st.txt

```diff
@@ -1,3 +1,73 @@
+in southern Italy.
+
+:: Authors
+Jules Verne
+
+:: References
+https://en.wikipedia.org/wiki/Journey_to_the_Center_of_the_Earth
+
+:: Ratings
+5 <paul>
+
+:: Choice Themes
+travel into the Earth [The whole story is about a trio of intrepid explorers who trek down old lava tubes to ultimately reach the center of the Earth.]
+
+:: Major Themes
+the thirst for knowledge [Professor Lidenbrock was driven by a desire to discover what lurked deep beneath the surface of the Earth. On the journey down, he comported himself in a single-minded and at times reckless manner so that he might satisfy his burning curiosity.]
+uncle and nephew [Professor Lidenbrock and his nephew Alex explored the bowels of the Earth together.]
+human vs. nature [Professor Lidenbrock, his nephew Alex, and their stoic Icelandic guide Hans contended with all types of dangers as they trekked down through an extinct volcano to the center of the Earth: the survived cave-ins, privation, tornadoes, a perilous voyage on a vast subsurface ocean, an encounter with a prehistoric sea creature, etc.]
+Hollow Earth [Professor Lidenbrock and his party pointedly proved wrong the prevailing theory that the Earth had a central source of intense heat by the simple expedient of trekking down the lava tubes of an extinct volcano all the way to the center of the planet.]
+geology [Professor Lidenbrock and his party commented upon numerous geological features on their trek down to the center of the Earth: Lidenbrock (and to a lesser extent Alex) relatively dated the geological strata they were passing through and examined their fossil contents, discussed about various minerals, entertained two opposing theories of what lay beneath the Earth, etc.]
+academia [Professor Lidenbrock was a stereotypical oblivious German academic who was interested solely in advancing science by disproving the prevailing theory that the Earth had an intense source of heat at its center.]
+the scientist profession [The trained geologist Professor Lidenbrock and his nephew protégé Alex were in their element as they explored the bowels of the Earth.]
+human character stereotype [The party guide Hans was portrayed as a stoic Scandinavian to the extreme.]
+risk taking vs. playing it safe [Professor Lidenbrock single-mindedly insisted that the party take big risks at every turn in an effort to reach the center of the Earth, while his nephew Alex constantly countered him with suggestions to take the cautious route - or else return home the same way they came.]
+
+:: Minor Themes
+cryptographic cypher [Professor Lidenbrock and Alex broke Arne Saknussemm's variant on a transposition cipher to read the 16th century Icelandic alchemist's encrypted message describing how he'd made it to the center of the Earth and back.]
+romantic love [Alex was mooning over his love interest, Gretchen.]
+female stereotype [Professor Lidenbrock goddaughter Gretchen was a stereotypical timid and obedient 19th century female.]
```

Comment:
I don't think we should use this when someone confirms to contemporary norms.

Reply:
I'm happy to take it out here.

But we should  clarify what the [https://themeontology.org/theme.php?name=female%20stereotype](female stereotype) theme is suppose to be. For example, [https://themeontology.org/theme.php?name=hysterical%20woman%20stereotype](hysterical woman stereotype) conforms to contemporary norms in 1950s stories, and it's been used in such stories without hesitation.

Also, for the record, it's not clear cut that the stereotype in question here conforms with the contemporary norms - just sayin'.

Reply:
Hysterical woman was never a "norm". It was a popular stereotype in the 50s that is now likely seen as a sexist thing to point out.

Reply:
And I mean if you think it was excessive then a "stereotypically timid woman" theme is conceivable. I just don't think that was the case in this 19:th century story seen from a 19:th century perspective. Also it's getting vague - we should focus on identifying stereotypes that are so on-the-nose that there can be little arguing about them.

Reply:
Let's take it out then.